io.elementary.mail.json

```diff
@@ -12,7 +12,9 @@
         "--socket=fallback-x11",
         "--socket=wayland",
         "--talk-name=org.gnome.evolution.dataserver.AddressBook10",
+        "--talk-name=org.gnome.evolution.dataserver.Calendar8",
         "--talk-name=org.gnome.evolution.dataserver.Sources5",
+        "--talk-name=org.gnome.evolution.dataserver.UserPrompter0",
```

Comment:
I don't think mail uses the Calendar and UserPrompter Services. don't need to expose them to the sandbox.